### PR TITLE
DOCS-9753 Slack Connector

### PR DESCRIPTION
**Pull request by dhirsch1**

### CODE REVIEW

File: slack/1.0/antora.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+nav:
+- modules/ROOT/nav.adoc
+asciidoc:
+  attributes:
+    page-component-desc: Enables access to the Slack platform.
+    page-connector-type: Connector
+    page-connector-level: Select
+    page-exchange-group-id: com.mulesoft.connectors
+    page-exchange-asset-id: mule-slack-connector
+    page-runtime-version: 4.1.1
```
Comment (adrian-sv) [inline slack/1.0/antora.yml]: min runtime version should be 4.2.1

File: slack/1.0/modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+* The Slack API
+* Anypoint Connectors
+* Mule runtime engine (Mule)
+* Elements and global elements in a Mule flow
+* Creating a Mule app using Anypoint Studio (Studio)
+
+Before creating an app, you must have:
+
+* Credentials to access the Slack target resource
+* Anypoint Studio version 7.1 or later
```
Comment (adrian-sv) [inline slack/1.0/modules/ROOT/pages/index.adoc]: this should correspond to the min mule runtime version 4.2.1

File: slack/1.0/modules/ROOT/pages/slack-connector-studio.adoc [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+You can reference a configuration file that contains ANT-style property placeholders (recommended), or you can enter your authorization credentials in the global configuration properties. For information about the benefits of using property placeholders and how to configure them, see xref:connectors::introduction/intro-connector-configuration-overview.adoc[Anypoint Connector Configuration].
++
+. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specified server.
+. Click *OK*.
+
+[[configure-other-fields]]
+== Configure Additional Connector Fields
+
+After you configure a global element for SAP Connector, configure the the other required fields for the connector.The required fields vary depending on which connector operation you use.
```
Comment (adrian-sv) [inline slack/1.0/modules/ROOT/pages/slack-connector-studio.adoc]: this should be Slack connector, not SAP

File: slack/1.0/modules/ROOT/pages/index.adoc [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+* New users, read:
+** xref:slack-connector-studio.adoc[Use Studio to Configure the Connector] to create your Mule app
+** xref:slack-connector-examples.adoc[Examples], which provide one or more use cases for the connector
+* Advanced users, read:
+** xref:slack-connector-xml-maven.adoc[XML and Maven Support]
+** xref:slack-connector-examples.adoc[Examples], which provides one or more use cases for the connector
+
+== Authentication Types
+
+Slack Connector uses the OAuth 2.0 authentication type, which uses access tokens provided by the authorization server to get access to protected data.
```
Comment (adrian-sv) [inline slack/1.0/modules/ROOT/pages/index.adoc:26]: should we have a section where it is explained how to get the token initially? like this one on Salesforce? https://docs.mulesoft.com/salesforce-connector/10.3/salesforce-connector-studio#oauth-2-0

**Review by sorinmare — DISMISSED** [review]
LGTM

File: slack/1.0/modules/ROOT/pages/index.adoc [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for Slack (Slack Connector) enables access to the Slack platform. This connector exposes all of the operations provided by Slack's REST API.
+
+For compatibility information and fixed issues, see the Slack Connector Release Notes.
+
+== Prerequisites
+
+To use this connector, you must be familiar with:
```
Comment (varunmaker) [inline slack/1.0/modules/ROOT/pages/index.adoc]: "To use this connector, you should be familiar with:"

File: slack/1.0/modules/ROOT/pages/index.adoc [hunk 6]
```diff
@@ -0,0 +1,38 @@
+= Slack Connector 1.0 - Mule 4
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for Slack (Slack Connector) enables access to the Slack platform. This connector exposes all of the operations provided by Slack's REST API.
```
Comment (varunmaker) [inline slack/1.0/modules/ROOT/pages/index.adoc]: "Anypoint Connector for Slack (Slack connector) give you access to the Slack platform. This connector exposes key operations provided by the Slack API. Please visit Slack's [official site](https://slack.com/help/articles/115004071768-What-is-Slack-) to learn more.

**Review by varunmaker — DISMISSED** [review]

**Pull request closed without merging**